Views/ProductTypes/List.cshtml

```diff
@@ -0,0 +1,20 @@
+@model IEnumerable<Bangazon.Models.Product>
+@using Bangazon.Models;
+
+@{
+    ViewData["Title"] = "Product Listing";
+}
+
+<p>
+    <a asp-action="Create">Sell Something</a>
+</p>
+
+@foreach (Product item in Model)
+{
+    <div class="row">
+        <div class="col-md-3">@Html.DisplayFor(modelItem => item.Name)</div>
+        <div class="col-md-3">@Html.DisplayFor(modelItem => item.Price)</div>
```

Comment:
Could add a `$` in front of `@Html.DisplayFor(modelItem => item.Price)` to make it more clear that this number being displayed is the price.